scripts/zones/Gustav_Tunnel/mobs/Nanoplasm.lua

```diff
@@ -4,23 +4,16 @@
 -----------------------------------
 package.loaded["scripts/zones/Gustav_Tunnel/TextIDs"] = nil;
 -----------------------------------
+
```

Comment:
uneeded newline. don't need to add or remove them as you touch files, I've a pr that mass eliminates the excess whitespace along with all the block comments but its waiting on bcnm branch merge coz @takhlaq is concerned about conflicts. Which is why I'll have to rebase and handle a bunch of conflicts myself.

Reply:
Just letting you know so you don't modify a bunch more of them. More conflicts :x